index.js

```diff
@@ -3,7 +3,7 @@ const os = require('os');
 
 const extractPathRegex = /\s+at.*(?:\(|\s)(.*)\)?/;
 const pathRegex = /^(?:(?:(?:node|(?:internal\/[\w/]*|.*node_modules\/(?:babel-polyfill|pirates)\/.*)?\w+)\.js:\d+:\d+)|native)/;
-const homeDir = os.homedir();
+const homeDir = typeof os.homedir === 'undefined' ? '' : os.homedir();
```

Comment:
Isn't this a `ReferenceError` when `os` is not a object?

Reply:
`os` should be an empty object when `browser.os = false` in package.json

Reply:
`os` should be an empty object when `browser.os = false` in package.json

Reply:
`os` should be an empty object when `browser.os = false` in package.json

Reply:
`os` should be an empty object when `browser.os = false` in package.json

Reply:
Indeed, I see it in https://github.com/defunctzombie/package-browser-field-spec#ignore-a-module. It's rather confusing that the value is to be set to `false` instead of something more sane like `{}`.

Reply:
I could still see it throwing that error for implementations that don't read `package.json`, for example when a browser loads the file via `<script type="module">`, so I'd advice changing this line to:

````js
os && typeof os.homedir === 'undefined' ? '' : os.homedir();
````